### PR TITLE
Pre-launch tasks

### PR DESCRIPTION
**Pull request by wolfadex**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 elm-stuff
 node_modules
-.direnv
\ No newline at end of file
+.direnv
+
```
Comment (miniBill) [inline .gitignore:4]: :nail_care: remove the empty line

File: README.md [hunk 2]
```diff
@@ -4,30 +4,39 @@ Generate an Elm SDK from an OpenAPI Spec (OAS).
...
+
 ## Notes
 
 - This was setup with Nix, and I'm not a Nix expert.
 - This currently compiles everything into a single file. This isn't neessarily bad as Elm has live-code-inclusion, but it does make it harder to work with. Maybe we should compile to multiple modules?
   - The GitHub Spec currently generates 120k+ loc, well below the ~1M loc limit for a module.
+
+## Thank you
+
+- [miniBill](https://github.com/miniBill/) for your **many** contributions
```
Comment (miniBill) [inline README.md:40]: :purple_heart:

File: generated/GitHub_v3_REST_API.elm
Comment (miniBill) [inline generated/GitHub_v3_REST_API.elm:1]: Should this file be part of the repo at all?

File: package.json [hunk 3]
```diff
@@ -1,21 +1,27 @@
...
   "description": "A tool for generating Elm SDKs from OpenAPI Specs.",
-  "main": "dist/elm-oas.js",
+  "main": "dist/elm-open-api.js",
+  "type": "module",
   "scripts": {
     "elm-pages": "elm-pages",
-    "dev": "elm-pages run script/src/Cli.elm",
-    "build": "elm-pages bundle-script script/src/Cli.elm --output dist/elm-oas.js"
+    "dev": "elm-pages run src/Cli.elm",
+    "build": "elm-pages bundle-script script/src/Cli.elm --output dist/elm-open-api.js --debug",
```
Comment (miniBill) [inline package.json:10]: Update the `build` command now that the scripts are in the root directory

**Comment by wolfadex** [comment]
Resolves #44

**Pull request merged**